model/validation.go

```diff
@@ -838,6 +838,55 @@ func ValidateParentAndDrain(drainTime, parentShutdown *duration.Duration) (errs
 				parentShutdownDuration.Seconds(), drainDuration.Seconds()))
 	}
 
+	if drainDuration > (time.Hour * 1) {
```

Comment:
I agree with you @ayj, from a stylistic standpoint the only reason I went with the defined variable is because this was the way it was done throughout the rest of the file - so I opted for uniformity